store-api-server/src/payment/service/payment.service.ts

```diff
@@ -571,26 +579,30 @@ WHERE nft_order_id = $1
   }
 
   async #createPaymentIntent({
-    user,
-    provider,
-    currency,
-    amountUnit,
-    clientIp,
-  }: ICreatePaymentIntent): Promise<PaymentIntentInternal> {
+      user,
+      provider,
+      currency,
+      amountUnit,
+      clientIp,
+    }: ICreatePaymentIntent,
+    nfts: NftEntity[]
```

Comment:
would be better to move this argument to the ICreatePaymentIntent object.

Reply:
Wasn't sure if changing the structure of ICreatePaymentIntent is safe and has no side effects. This is the safest solution I can see for a short term patch. Given the risk and time constraints might be better to leave it like this for now?

Reply:
Fair enough. In principle this object is just used for this function. It's done this way because there are so many arguments, having them named is less error prone when defining the arguments on the function call locations.

But I understand your decision here and agree. Lets let this be.